### ISSUE TITLE
Improvement for k8s.io/docs/setup/best-practices/cluster-large/

### ISSUE THREAD

**Issue by skamchiline** [kubernetes/website#28068]
This page used to contaon exact recommendations on the master node types to use for different clusters. Lots of internet resources are still referring to this page as a source of the info (this page as an example: https://learnk8s.io/kubernetes-node-size). But the recommended master node types are removed from here. 

I'd really like to have this info here.

Thank you,

**Comment by sftim** [comment]
The text you're thinking of was:
> On GCE/Google Kubernetes Engine, and AWS, kube-up automatically configures the proper VM size for your master depending on the number of nodes in your cluster. On other providers, you will need to configure it manually. For reference, the sizes we use on GCE are
>
>    1-5 nodes: n1-standard-1
>    6-10 nodes: n1-standard-2
>    11-100 nodes: n1-standard-4
>    101-250 nodes: n1-standard-8
>    251-500 nodes: n1-standard-16
>    more than 500 nodes: n1-standard-32
>
> And the sizes we use on AWS are
>
>    1-5 nodes: m3.medium
>    6-10 nodes: m3.large
>    11-100 nodes: m3.xlarge
>    101-250 nodes: m3.2xlarge
>    251-500 nodes: c4.4xlarge
>    more than 500 nodes: c4.8xlarge

Those recommendations are:
- no longer current
- tied to specific, third party providers

If cloud providers want to publish their own recommendations, the Kubernetes project has no objection. We wouldn't reintroduce that list because it's difficult to cover the many cloud providers that Kubernetes can run on, and because it's out of line with our current [content guide](https://kubernetes.io/docs/contribute/style/content-guide/) (which was one reason I proposed removing it).

**Comment by skamchiline** [comment]
Yes, Tim. Thank you for the explanation. I think the page would benefit if it would contain at least generic recommendations to the master nodes in terms of CPU cores and mem size. Otherwise sysadmins are pretty blind on what to use and have to run into  performance issues before finalizing their configurations. 

Thank you, TIm, again.

**Comment by sftim** [comment]
/retitle Add generic sizing recommendations to Best Practices setup pages
/language en
/kind feature

**Comment by sftim** [comment]
@kubernetes/sig-scalability would anyone from the SIG be willing to triage this and maybe help move the work forward?

**Comment by mm4tt** [comment]
/assign @marseel 

Note that Marcel is OOO this week, but he should be able to help with that once he is back.

**Comment by wojtek-t** [comment]
We should update https://github.com/kubernetes/community/blob/master/sig-scalability/configs-and-limits/provider-configs.md and link it from there [though not in the current format...]

/remove-lifecycle stale

**Comment by sftim** [comment]
As a contributor mainly to SIG Docs, I won't mark this as frozen. However, SIG Scalability folks: you are welcome to triage this as Accepted, and then also to freeze it.

**Comment by sftim** [comment]
Repeating https://github.com/kubernetes/website/issues/28068#issuecomment-856953198:
> @kubernetes/sig-scalability would anyone from the SIG be willing to triage this and maybe help move the work forward?

**Comment by divya-mohan0209** [comment]
> We should update https://github.com/kubernetes/community/blob/master/sig-scalability/configs-and-limits/provider-configs.md and link it from there [though not in the current format...]
> 
> /remove-lifecycle stale

@wojtek-t Would this still be your recommendation as it stands today? If yes, we can open it up to contributors for edits.

**Comment by wojtek-t** [comment]
> @wojtek-t Would this still be your recommendation as it stands today? If yes, we can open it up to contributors for edits.

Yeah - I think it's still a valid one.

**Comment by divya-mohan0209** [comment]
Thank you so much for confirming! 
@marseel would you still be interested in contributing?

**Comment by divya-mohan0209** [comment]
@kubernetes/sig-scalability-leads : Please could you advise if anyone could help triage this PR and move this forward?
